pkg/repository/prisma/dbsqlc/step_runs.sql

```diff
@@ -664,6 +664,7 @@ WITH step_runs_on_inactive_workers AS (
     WHERE
         w."tenantId" = @tenantId::uuid
         AND w."lastHeartbeatAt" < NOW() - INTERVAL '30 seconds'
+        AND sr."status" in ('PENDING','PENDING_ASSIGNMENT','ASSIGNED','RUNNING','CANCELLING' )
```

Comment:
is this now required due to partitioning?
why is this change here?

Reply:
Probably more important before partitioning actually  - it's probably redundant now because we partition on that column